xbmc/cores/AudioEngine/Interfaces/AEStream.h

```diff
@@ -13,6 +13,7 @@
 
 extern "C" {
 #include <libavcodec/avcodec.h>
+#include <libavutil/channel_layout.h>
```

Comment:
Is there any particular reason why you added it to the header?
Is it sufficient to put it in the cpp file where it's needed?